uqcsbot/scripts/emojify.py

```diff
@@ -0,0 +1,113 @@
+                           "card-queen-hearts", "card-queen-spades"]),
+                   "quora", "quantium"]
+    master['R'] = ["r-project", "rust", "redroom", "registered"]
+    master['S'] = ["s", "skedulo", "stanford", "stripe_s", "sublime", "tetris_s"]
+    master['T'] = ["tanda", choice(["telstra", "telstra-pink"]),
+                   "tesla", "tetris_t", "torchwood", "tumblr"]
+    master['U'] = ["uber", "uqu", "the_horns"]
+    master['V'] = ["vim", "vue", "vuetify", "v"]
+    master['W'] = ["office_word", "washio", "wesfarmers", "westpac",
+                   "weyland_consortium", "wikipedia_w", "woolworths"]
+    master['X'] = ["atlassian_old", "aginicx", "sonarr", "x-files", "xbox", "x", "flag-scotland"]
+    master['Y'] = ["hackernews"]
+    master['Z'] = ["tetris_z"]
+
+    master['0'] = ["chrome", "suncorp", "disney_zero", "firefox", "mars", choice(["dvd", "cd"])]
+    master['1'] = ["techone", "testtube", "thonk_ping", "first_place_medal"]
+    master['2'] = [choice(["card-2-clubs", "card-2-diamonds", "card-2-hearts", "card-2-spades"]),
+                   "second_place_medal"]
+    master['3'] = [choice(["card-3-clubs", "card-3-diamonds", "card-3-hearts", "card-3-spades"]),
+                   "css", "third_place_medal"]
+    master['4'] = [choice(["card-4-clubs", "card-4-diamonds", "card-4-hearts"]), "card-4-spades"]
+    master['5'] = [choice(["card-5-clubs", "card-5-diamonds", "card-5-hearts", "card-5-spades"]),
+                   "html"]
+    master['6'] = [choice(["card-6-clubs", "card-6-diamonds", "card-6-hearts", "card-6-spades"])]
+    master['7'] = [choice(["card-7-clubs", "card-7-diamonds", "card-7-hearts", "card-7-spades"])]
+    master['8'] = [choice(["card-8-clubs", "card-8-diamonds", "card-8-hearts", "card-8-spades"]),
+                   "8ball"]
+    master['9'] = [choice(["card-9-clubs", "card-9-diamonds", "card-9-hearts", "card-9-spades"])]
+
+    master[' '] = ["whitespace"]
+
+    master['@'] = ["whip"]
+    master['#'] = ["slack_old", "csharp"]
+    master['$'] = ["thonk_money", "moneybag"]
+    master['^'] = ["this", "typographical_carrot", "arrow_up"]
+    master['&'] = ["ampersand", "dnd"]
+    master['*'] = ["day", "nab", "youtried", "msn_star", "rune_prayer"]
+    master['-'] = ["no_entry"]
+    master['+'] = ["tf2_medic", "flag-ch", "flag-england"]
+
+    master['"'] = [choice(["ldquo", "rdquo"]), "pig_nose"]
+    master['>'] = ["accenture", "implying", "plex", "powershell"]
+    master['/'] = ["slash"]
+
+    # master['Α'] = ["alpha"]
+    # master['Β'] = ["beta"]
+    # master['Λ'] = ["halflife", "haskell", "lambda", "racket"]
+    # master['Σ'] = ["polymathian"]
+
+    # master['…'] = ["lastpass"]
+    # master['€'] = ["martian_euro"]
+    # master['√'] = ["sqrt"]
+    # master['∞'] = ["arduino", "visualstudio"]
+    # master['∴'] = ["julia"]
+
+    text = ""
+    if command.has_arg():
+        text = command.arg.upper()
+    text = text.replace("&AMP;", "&")
```

Comment:
Why are these replacements necessary? Probably explain in a comment, too.

Reply:
`&` converts to `&AMP;`, `>` converts to `&GT;`. This reverse that. Added comment.